app/assets/javascripts/retro.js

```diff
@@ -1,20 +1,18 @@
-$(document).ready(function() {
+$(document).ready(() => {
   // Get HTML elements
   let startNewRetroButton = $("#start-new-button");
   let timeText = $("#time");
 
-  let timeNow;
-
   // Timer values
 
   const postingTime = 1000 * 60 * 60;
   const postingTimeWarning = 1000 * 60 * 50;
 
-  function checkTimeLeft() {
-    if (timeNow === postTimer) {
-      return false;
-    } else if (timeNow >= 1000 * 60 * 50) {
-      timeText.style.color = "#FF0000";
+  // Aux functions
+
+  function checkTimeLeft(time, timeMax) {
+    if (time >= timeMax) {
+      timeText.css({ "color": "#FF0000" });
```

Comment:
With jQuery it is possible to [add a class name](https://api.jquery.com/addclass/) to the class list of an element.
I think that it could be more organized and maintainable if you keep css properties inside ids or classes rather than add style properties to an element directly.
What do you think about that?